src/main/java/htsjdk/samtools/util/GZIIndex.java

```diff
@@ -221,31 +220,33 @@ public String toString() {
     /**
      * Writes this index into the requested path.
      *
+     * NOTE: This method will close out the provided output stream when it finishes writing the index
+     *
      * @param output the output file.
      *
      * @throws IOException if an I/O error occurs.
      */
-    public void writeIndex(final Path output) throws IOException {
+    void writeIndex(final OutputStream output) throws IOException {
```

Comment:
We shouldn't just remove an existing public method without at least deprecating it first. However, it seems unnecessary to do when you could just add the stream overload and delegate to that.

Reply:
Whoops, this was accidental. I agree keeping the old api should be fine (despite it not being overwhelmingly old).